src/ui/components/ActionPanel/ActionPanel.tsx

```diff
@@ -57,7 +57,7 @@ type OwnProps = {
     };
     renderRevisionItemActions?: (item: GetRevisionsEntry, currentRevId: string) => React.ReactNode;
     wrapperRef?: React.Ref<HTMLDivElement>;
-    pageOffset?: string;
+    styles?: React.CSSProperties;
```

Comment:
Here it would be great to maintain continuity with the react naming: ~~styles~~ -> `style`